### PR TITLE
I added a lot...

### PR DESCRIPTION
**Pull request by AZO234**
I added a lot.

- Skills
- ItemDetail class
- Corpuse and Golem Item
- color by use colorama

### CODE REVIEW

**Comment by oaken-source** [comment]
Hey, wow, thank you. That is a lot of contribution for a single PR.
It would be easier for me to review if you gave the commits a more descriptive commit message though. just having it say 'fix' does not really help :)

**Comment by oaken-source** [comment]
Also, I hope you're allright, despite the current flooding in Hiroshima. Best Regards!

File: pyd2s/questdata.py [hunk 1]
```diff
@@ -154,15 +154,15 @@ def set_act3_lamesen(self, difficulty, value):
             self._buffer[self._getoffset(difficulty) + 42] = 0x00
             self._buffer[self._getoffset(difficulty) + 43] = 0x00
 
-    def get_act4_isual(self, difficulty):
```
Comment (oaken-source) [inline pyd2s/questdata.py]: this commit should probably be squashed into 37feb19588b821ab2cfca02d6f4773761dbd36aa

File: pyd2s/questdata.py [hunk 2]
```diff
@@ -42,6 +42,7 @@ def get_act1_resetstatus(self, difficulty):
         '''
         Get Reset status end flag
         '''
+#        print("0x{:04X}".format((self._buffer[self._getoffset(difficulty) + 83] << 8) | self._buffer[self._getoffset(difficulty) + 82]))
```
Comment (oaken-source) [inline pyd2s/questdata.py]: this commit needs a better subject - what does it 'fix'? also, I think the commented print() lines should go.

File: pyd2s/__init__.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 from pyd2s.savebuffer import SaveBuffer
 from pyd2s.character import Character
 from pyd2s.mercenary import Mercenary
-from pyd2s.item import Item
```
Comment (oaken-source) [inline pyd2s/__init__.py:11]: this should probably be 'from ... import Item, ...' instead of a star import

File: sample/pyd2s_stat.py [hunk 4]
```diff
@@ -367,20 +367,20 @@
     print(temp)
     temp = "  "
     if o_itemdata.getquality == 1:
-        temp += "[LQ]"
```
Comment (oaken-source) [inline sample/pyd2s_stat.py]: the 'fix' commits can probably all be squashed.

File: pyd2s/questdata.py (hunk 2 above)
Comment (AZO234) [reply pyd2s/questdata.py]: These flags are summarized in Act, so they have the same value displayed.

File: pyd2s/__init__.py (hunk 3 above)
Comment (AZO234) [reply pyd2s/__init__.py:11]: I wanted to use the isarmors function, the isweapons function, etc,
but I modify it to the property of the item class.

Thanks.

**Comment by AZO234** [comment]
Yeah, flooding in Hiroshima is a terrible situation.
Fortunately, there was no damage to my house.
Thank you for your concern.

**Comment by oaken-source** [comment]
Hey, it's been a while. I think I covered most of the things in this PR, but feel free to take a look if anything is missing!

**Pull request closed without merging**